docker/shared_steps/install_php_extensions.sh

```diff
@@ -10,6 +10,17 @@ php_install_gd() {
   eatmydata docker-php-ext-install -j$COMPILE_JOBS $ext
 }
 
+php_install_ldap() {
+  local php=$(which php)
+  local php_version=$($php --version | head -n1 | cut -d " " -f 2 | cut -d . -f 1,2)
+  if [ $php_version = "7.0" ]; then
+    install_packages "libldb-dev"
```

Comment:
This is a dev package right? Please mark dev packages (only required during compilation of the module) as temporary via `install_packages --build libldb-dev`. So the package gets removed after the build finishes. This helps reducing the actual size of the resulting image size.